<?xml version="1.0"?>
<launch>
    <include file="$(find assignment2_part1)/launch/sim_w1.launch" />
    <param name="des_pos_x" value= "0.0" />
    <param name="des_pos_y" value= "1.0" />
    <node pkg="assignment2_part1" type="wall_follow_service.py" name="wall_follower" />
    <node pkg="assignment2_part1" type="go_to_point_service.py" name="go_to_point"  />
    <node pkg="assignment2_part1" type="bug_as.py" name="bug_action_service" output="screen" />
    <node pkg="assignment2_part1" type="assignment.py" name="Assignment_node" output="screen" launch-prefix="xterm -e"/>
    <node pkg="assignment2_part1" type="user_input.py" name="UserInput_node" output="screen" launch-prefix="xterm -e"/>
</launch>
